packages/js/email-editor/src/components/block-editor/editor.tsx

```diff
@@ -53,27 +54,35 @@ export function InnerEditor( {
 		'post-only'
 	);
 
-	const { post, template, isFullscreenEnabled } = useSelect(
-		( select ) => {
-			const { getEntityRecord } = select( coreStore );
-			const { getEditedPostTemplate } = select( storeName );
-			const postObject = getEntityRecord(
-				'postType',
+	// isFullScreenForced – comes from settings and cannot be changed by the user
+	// isFullscreenEnabled – indicates if a user has enabled fullscreen mode
+	const { post, template, isFullScreenForced, isFullscreenEnabled } =
+		useSelect(
+			( select ) => {
+				const { getEntityRecord } = select( coreStore );
+				const { getEditedPostTemplate } = select( storeName );
+				const postObject = getEntityRecord(
+					'postType',
+					currentPost.postType,
+					currentPost.postId
+				);
+				return {
+					template:
+						currentPost.postType !== 'wp_template'
+							? getEditedPostTemplate()
+							: null,
+					post: postObject,
+					isFullscreenEnabled:
+						select( storeName ).isFeatureActive( 'fullscreenMode' ),
+					isFullScreenForced: settings.isFullScreenForced,
```

Comment:
`isFullScreenForced` doesn't need to be in `useSelect`.
It is not fetched from the store, but just passed via the `settings` prop so it can be accessed directly.
So you can do just `const { isFullScreenForced } = settings;` outside the useSelect.

Reply:
I will update the code. But can you explain the reason for moving it outside? Is there a downside to doing it this way? Does it cause any performance issues or any other issues?